tests/test_url_handler.py

```diff
@@ -81,7 +81,7 @@ def test_metadata_list_includes_valueset_values(self):
         results = url_handler.get_metadata(project)
 
         for _, value_iter in results:
-            assert list(value_iter).sort() == vals.sort()
+            assert all(value in vals for value in list(value_iter))
```

Comment:
Should we also be testing the reverse, that everything in `vals` appears in `value_iter`? Otherwise, couldn't we have `vals = [1, 2, 3, 4, 5]` and a `value_iter = [1, 2, 3]` that would pass this test when we'd want it to fail?

Reply:
Since we are setting up `vals` to be in the test of length 3, could change it to:
```
            value_iter = list(value_iter)
            assert all(value in vals for value in value_iter)
            assert len(value_iter) == 3
```